requirements.txt

```diff
@@ -3,3 +3,5 @@ sanic-gunicorn==0.1.2
 aiohttp==2.2.5
 sqlalchemy==1.1.14
 psycopg2==2.7.3.1
+geoalchemy2==0.4.0
+shapely==1.6.1
```

Comment:
Should probably be a test dependency , not runtime dependency, in line with above comment re: `wkt_to_location`